ViewGeneratorCore/build/ViewGeneratorCore.targets

```diff
@@ -36,13 +36,33 @@
       <__ViewGeneratorCoreNodeJsExe>@(_ViewGeneratorCoreNodeJsExeAfterCheckedPaths);</__ViewGeneratorCoreNodeJsExe>
       <ViewGeneratorCoreNodeJsExe>$(__ViewGeneratorCoreNodeJsExe.Substring(0, $(__ViewGeneratorCoreNodeJsExe.IndexOf(';'))))</ViewGeneratorCoreNodeJsExe>
     </PropertyGroup>
+
+      <PropertyGroup>
```

Comment:
have you tested this in ss?

Reply:
Yes, this seems to be working fine. Do you have any concern about this?

Reply:
the regex scares me a bit

Reply:
I understand, any suggestions? new config file?

I don't know if we can parse JSON on msbuild without external packages, that way we could access the input key without regex ..